<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SelectFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectB">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="binaryImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>720</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="sliderPosition">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="threshold">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>720</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdjustImage">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HistogramEqualization">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Hist Equalization</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>541</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="adjLayout"/>
   </widget>
   <widget class="QPushButton" name="Contrast">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>500</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Difference</string>
    </property>
   </widget>
   <widget class="QSlider" name="Brightness">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>730</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-256</number>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ContrastSlider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>770</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="brightness">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>730</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="contrastvalue">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>760</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayScale">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="grayLevel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Sizevalue">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>42</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="enlargebutton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>610</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enlarge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shrinkbutton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>660</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Shrink</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
